org.gnome.Calendar.json

```diff
@@ -25,8 +25,6 @@
         "--env=DCONF_USER_CONFIG_DIR=.config/dconf"
     ],
     "build-options" : {
-        "cflags" : "-O2 -g",
-        "cxxflags" : "-O2 -g",
         "env" : {
             "V" : "1"
```

Comment:
I’d remove this bit too, in my experience it just makes the build output uglier.

Reply:
But that’s just my opinion, it’s up to @GeorgesStavracas in the end.

Reply:
Remove that!